### ISSUE TITLE
Placeholders not working

### ISSUE THREAD

**Issue by oscartbeaumont** [alexflint/go-arg#98]
Placeholders are not working for me. I run the following example code from the README.
```bash
var args struct {
	Input    string   `arg:"positional" placeholder:"SRC"`
	Output   []string `arg:"positional" placeholder:"DST"`
	Optimize int      `arg:"-O" help:"optimization level" placeholder:"LEVEL"`
	MaxJobs  int      `arg:"-j" help:"maximum number of simultaneous jobs" placeholder:"N"`
}
arg.MustParse(&args)
```
and the output I get it as follows.
```bash
Usage: mattrax [--optimize OPTIMIZE] [--maxjobs MAXJOBS] INPUT [OUTPUT [OUTPUT ...]]

Positional arguments:
  INPUT
  OUTPUT

Options:
  --optimize OPTIMIZE, -O OPTIMIZE
                         optimization level
  --maxjobs MAXJOBS, -j MAXJOBS
                         maximum number of simultaneous jobs
  --help, -h             display this help and exit
```
My understanding is I should get the output
```bash
Usage: mattrax [--optimize LEVEL] [--maxjobs N] SRC [DST [DST ...]]

Positional arguments:
  SRC
  DST

Options:
  --optimize LEVEL, -O LEVEL
                         optimization level
  --maxjobs N, -j N      maximum number of simultaneous jobs
  --help, -h             display this help and exit
```

**Comment by doczi-dominik** [comment]
It seems that placeholders only work when there is not a `go.mod` (or `go.sum`) file inside the package directory.

**Steps to reproduce issue:**

1. Create a new directory (e.g.: `testing`) in `$GOPATH/src` and `cd` to it
2. Create a `main.go` file with the example.
3. `go run main.go -h` **--> Output is correct: Args use their placeholders**
4. Run `go mod init github.com/x/testing`
5. `go run main.go -h` **--> Output is wrong: Args use their uppercase names**

**Comment by runeimp** [comment]
This is happening to me as well. With `go version go1.13.7 darwin/amd64`

**Comment by florentsolt** [comment]
I had the same issue, it was a version issue. I mean I had the wrong version installed.
Try with the master version `go get github.com/alexflint/go-arg@master`
It did fix the missing placeholders.

**Comment by alexflint** [comment]
Yes this feature was introduced recently. Apologies @oscartbeaumont for not documenting this more clearly. I've now released this feature in `v1.3.0` and #108 documents that this feature requires this version. Thanks for the clear and thorough bug report.
